v2/lints/cabf_br/lint_ext_nc_intersects_reserved_ip.go

```diff
@@ -0,0 +1,63 @@
+package cabf_br
+
+/*
+ * ZLint Copyright 2020 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/************************************************
+BRs: 7.1.5
+(b) For each iPAddress range in permittedSubtrees, the CA MUST confirm that the
+Applicant has been assigned the iPAddress range or has been authorized by the
+assigner to act on the assignee's behalf.
+
+BRs: 7.1.4.2.1
+CAs SHALL NOT issue certificates with a subjectAlternativeName extension or
+Subject commonName field containing a Reserved IP Address or Internal Name.
+************************************************/
+
+import (
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/v2/lint"
+	"github.com/zmap/zlint/v2/util"
+)
+
+type NCReservedIPNet struct{}
+
+func (l *NCReservedIPNet) Initialize() error {
+	return nil
+}
+
+func (l *NCReservedIPNet) CheckApplies(c *x509.Certificate) bool {
+	return c.NotAfter.After(util.NoReservedIP) && util.IsExtInCert(c, util.NameConstOID)
```

Comment:
Is this the date after which CAs cannot issue certificates or the date after which certificates can no longer contain reserved IPs?

Reply:
The latter I believe, I wasn't entirely sure if it's the best choice, but I thought it made sense to keep in line with the `e_ext_san_contains_reserved_ip` lint.